lib/src/list_of.ml

```diff
@@ -342,7 +343,8 @@ module Make (Char: Api.BASIC_CHARACTER) :
     let output chan l =
       List.fold_left l ~init:(Model.return ()) ~f:(fun prev_m c ->
           prev_m >>= fun () ->
-          Model.output chan (Char.to_native_string c))
+          (* TODO: Safe to call Bytes.unsafe_to_string? *)
+          Model.output chan (Char.to_native_bytes c |> Bytes.to_string))
```

Comment:
`Model.output` is user-provided so I don't think it would be 100% safe to use `Bytes.unsafe_to_string`.